### DISCUSSION TITLE
Consumer Key & Secret API Kaskus

### DISCUSSION THREAD

**Discussion by superdiazzz** [farizdotid/DAFTAR-API-LOKAL-INDONESIA#318] [General]
ane biingung cara ngedapeting consumer key dan consumer secret key nya gan...

**Comment by ryandewantoro** [comment]
up lagi

**Comment by farizdotid** [comment]
@ryandewantoro Sama euy ane juga ga nemu gan. Apa perlu kontak CSnya langsung untuk tanya portal developernya?

**Comment by ryandewantoro** [comment]
@farizdotid iya nih perlu banget

**Comment by sooluh** [comment]
bisa dilihat di [sini](https://gist.github.com/repodevs/83d727cc1d093b1e857ca4970a49b8c7#file-kaskus-py-L18), sepertinya Kaskus tidak menyediakan API secara official, mungkin bisa dilakukan reverse engineering pada aplikasi mobile-nya, tapi secara hukum ini ilegal.
